decentraland/unity-explorer
# fix: Header value contains invalid characters

## What does this PR change?

Fixes most probably all of these sentry issues:
https://decentraland.sentry.io/issues/?environment=production&project=4506075736047616&query=is%3Aunresolved%20%22Header%20value%20contains%20invalid%20characters%22&referrer=issue-list&sort=date&statsPeriod=120d&viewId=default0
and the related ticket: #2794 


Studying the issues I noticed they happened to turkish users. Which led me into a rabbit hole until I found an issue when converting the payload of the webRequestInfo into lowercase. In turkish, apparently the lowercase of `I` is not `i` but `ı`, which caused this error to trigger :hideThePain.
Now it should be fixed also for any other language that could introduce similar errors.

...

## How to test the changes?

Just make sure the game works as normal. You can try setting the OS language to turkish and testing the game as well. All should work equally in both cases.
